app/src/main/java/net/gsantner/markor/activity/DocumentEditFragment.java

```diff
@@ -742,29 +736,14 @@ public void setUserVisibleHint(boolean isVisibleToUser) {
         }
     }
 
-    @Override
-    public void onFragmentFirstTimeVisible() {
-        final boolean initPreview = _appSettings.getDocumentPreviewState(_document.getPath());
-        if (_savedInstanceState == null || !_savedInstanceState.containsKey(SAVESTATE_CURSOR_POS) && _hlEditor.length() > 0) {
-            int lastPos;
-            if (_document != null && (lastPos = _appSettings.getLastEditPositionChar(_document.getFile())) >= 0 && lastPos <= _hlEditor.length()) {
-                if (!initPreview) {
-                    _hlEditor.requestFocus();
-                }
-                _hlEditor.setSelection(lastPos);
-            } else if (_appSettings.isEditorStartOnBotttom()) {
-                if (!initPreview) {
-                    _hlEditor.requestFocus();
-                }
-                _hlEditor.setSelection(_hlEditor.length());
-            }
-        }
```

Comment:
All of this logic has been simplified and moved directly into `onCreate`

Reply:
hm, still everything works as expected when using some other applicatoins and switching back to markor? Especially on the main screen